### DISCUSSION TITLE
How to hide Search icon in AutoComplete

### DISCUSSION THREAD

**Discussion by markarnolditpro** [microsoft/fluentui-blazor#1563] [Q&A]
How best to hide the search icon shown by default in AutoComplete?  

Looks like we could just add a parameter [here](https://github.com/microsoft/fluentui-blazor/blob/2aafb48e4de1c3432c2c59f525a3cd856f71a613/src/Core/Components/List/FluentAutocomplete.razor.cs)
```
[Parameter]   
public bool ShowSearchIcon { get; set; } = true;
```
and evaluate it [here](https://github.com/microsoft/fluentui-blazor/blob/2aafb48e4de1c3432c2c59f525a3cd856f71a613/src/Core/Components/List/FluentAutocomplete.razor#L78)
```
else if (ShowSearchIcon) { ... }
```

I can submit a PR if there's interest. 

Thanks!

**Comment by dvoituron** [comment]
I don't think you can just add a test like this because another part of the code calculates the size of the available width (including the size of the icon).

But, to be more customizable, we could add two nullable properties `IconSearch` and `IconDismiss`.

**Comment by markarnolditpro** [comment]
Sounds good.  To confirm, the defaults on these properties would be the current icons, however one could provide a different icon, or null, to control what's shown?
